portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -59,23 +52,23 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    List<Comment> comments = new ArrayList<>();
     Query commentsQuery = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     List<Entity> results = datastore.prepare(commentsQuery)
                           .asList(FetchOptions.Builder.withLimit(MAX_COMMENTS_FETCHED));
-
+    
     for (Entity entity : results) {
       long id = entity.getKey().getId();
       String content = (String) entity.getProperty("content");
       long timestamp = (long) entity.getProperty("timestamp");
-
       Comment comment = new Comment(id, content, timestamp);
       comments.add(comment);
     }
 
     Gson gson = new Gson();
```

Comment:
supernit: instead of initializing a new Gson object every time this function is called, you can initialize it in init() instead like you did w/ comments and reuse it.